modules/combined/tutorials/introduction/step01.i

```diff
@@ -0,0 +1,174 @@
+#
+# A first attempt at thermo mechanical contact
+# https://mooseframework.inl.gov/modules/combined/tutorials/introduction/step01.html
+#
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  block = 0
+[]
+
+[Mesh]
+  [generated1]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 5
+    ny = 15
+    xmin = -0.6
+    xmax = -0.1
+    ymax = 5
+    bias_y = 0.9
+    boundary_name_prefix = pillar1
+  []
+  [generated2]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 6
+    ny = 15
+    xmin = 0.1
+    xmax = 0.6
+    ymax = 4.999
+    bias_y = 0.9
+    boundary_name_prefix = pillar2
+    boundary_id_offset = 4
+  []
+  [collect_meshes]
+    type = MeshCollectionGenerator
+    inputs = 'generated1 generated2'
+  []
+
+  patch_update_strategy = iteration
+[]
+
+[Variables]
+  # temperature field variable
+  [T]
+    # initialize to an average temperature
+    initial_condition = 50
+  []
+  # temperature lagrange multiplier
+  [Tlm]
+    block = 'pillars_secondary_subdomain'
+  []
+[]
```

Comment:
It can be a bit odd to create a finite element model without mention to the finite element used. I understand it's whatever `FIRST` `LAGRANGE` gives you here, since those are default. It may be good to mention how this works in MOOSE during the first step.